src/components/audio-call-page/audio-call-page.css

```diff
@@ -133,3 +136,19 @@
     transform: translateY(0px);
   }
 }
+
+@media (max-width: 390px) {
+
+  .card-img-top {
+    display: none;
+  }
+
+  .current-word {
+    height: 200px;
+  }
+
+  .row-buttons-choice-wrapper {
+    min-height: 230px;
+  }
+
+}
```

Comment:
Add empty line at the end of the file

Reply:
done, [e99d0b1](https://github.com/Jeneko/rslang/pull/109/commits/e99d0b18e6ee52a805325649a2b93aa1295a96cc)